plugins/persisted-scope/Cargo.toml

```diff
@@ -1,22 +1,38 @@
 [package]
 name = "tauri-plugin-persisted-scope"
-version = "0.1.0"
+version = "0.1.1"
 description = "Save filesystem and asset scopes and restore them when the app is reopened."
-authors.workspace = true
-license.workspace = true
-edition.workspace = true
-rust-version.workspace = true
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+  [package.authors]
+  workspace = true
+
+  [package.license]
+  workspace = true
+
+  [package.edition]
+  workspace = true
+
+  [package.rust-version]
+  workspace = true
 
 [dependencies]
-serde.workspace = true
-serde_json.workspace = true
-tauri.workspace = true
-log.workspace = true
-thiserror.workspace = true
 aho-corasick = "0.7"
 bincode = "1"
 
+  [dependencies.serde]
+  workspace = true
+
+  [dependencies.serde_json]
+  workspace = true
+
+  [dependencies.tauri]
+  workspace = true
+
+  [dependencies.log]
+  workspace = true
+
+  [dependencies.thiserror]
+  workspace = true
+
```

Comment:
@jbolda Do you think this is fixable (in a sane/easy way)? It's not the end of the world but i'm not much of a fan either 😅 

p.s. i also tried it locally using covector from git because of [this PR](https://github.com/jbolda/covector/pull/268) which doesn't seem to be published yet but the result was the same.

Reply:
Rust has a nice library to use toml, but JS doesn't AFAIK :(

Reply:
So the easiest way is probably napi-rs the Rust crate and use it on covector

Reply:
Yeah, i didn't find any good packages either :(